src/queries.ts

```diff
@@ -15,6 +16,26 @@ namespace Queries {
     return nodes;
   };
 
+  export function getReturnType(functionNode: n.FunctionDeclaration): string {
+    let result;
+
+    //handle type reference as array for example
+    if (get(functionNode, 'returnType.typeAnnotation.type') === 'TSTypeReference') {
+      result = get(functionNode, 'returnType.typeAnnotation.typeName.name');
+    }
+
+    //handle annotation like string for example
+    if (!result) {
+      result = get(functionNode, 'returnType.typeAnnotation.type');
```

Comment:
You can create a constans.ts or strings.json file in which to put read-only(const) variable

Reply:
This string currently is used in one place. Ergo is no reason to put this in constant as long as there is no repetition.

Reply:
line 23 have this same string 'returnType.typeAnnotation.type'. 

The idea of ​​creating a file is that when you create a const in another file, first you can check if it already exists, then there is a small probability of a mistake in writing the same string a second time, and possibly you can use it a second time if necessary. In addition, such a file is a box for read-only variables.